src/docs/sdk/check-ins.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Check-Ins
+sidebar_order: 20
+---
+
+## Check-In Payload
+
+A check-in is an item in an envelope called `check_in`. It consists
+of a JSON payload that looks roughly like this:
+
+```json
+{
+  "check_in_id": "83a7c03ed0a04e1b97e2e3b18d38f244",
+  "monitor_slug": "b7645b8e-b47d-4398-be9a-d16b0dac31cb",
+  "status": "in_progress",
+  "duration": 10.0,
+  "release": "1.0.0",
+  "environment": "production"
+}
+```
+The following fields exist:
+
+`check_in_id`
+
+: **String, required**. Check-In ID (unique and client generated).
+
+`monitor_slug`
+
+: **String, required**. The distinct slug of the monitor.
+
+`status`
+
+: **String, required**. The status of the check-in. Can be one of the following:
+  - `in_progress`: The check-in has started.
+  - `ok`: The check-in has completed successfully.
+  - `error`: The check-in has failed.
+
+`duration`
+
+: _Number, optional_. The duration of the check-in. Will only take effect if the
```

Comment:
```suggestion
: _Number, optional_. The duration of the check-in in milliseconds. Will only take effect if the
```

Reply:
so even more important to add the resolution :-) 
Is it only ints or can it be a float?